tests/functions_tests/test_edge_extraction.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import torch
+from torch_dftd.functions.edge_extraction import calc_neighbor_by_ase, calc_neighbor_by_pymatgen
+
+
+def test_calc_neighbor_equivalent():
+    n_nodes = 5
+    pbc = torch.tensor([True, True, True])
+    cell = torch.randn((3, 3))
```

Comment:
It seems `cell` could be negative volume. I think that is OK. However, is it also possible that the volume is almost zero, for example, one of the lattice vector is almost zero or the rank of `cell` is less than 3?
In my understanding, the number of neighbors could be arbitrary large in those cases.